src/main/java/com/ericsson/eiffel/remrem/publish/service/SendResult.java

```diff
@@ -1,14 +1,22 @@
 package com.ericsson.eiffel.remrem.publish.service;
 
```

Comment:
For achieve output format, we required to maintain two class. Please suggest if I wrong.